recipes/pystreamer/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pystreamer" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "200e910b273570aaf93aec076a399dfec413afa6bfc049943a0f2599c22fa937" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build: {}
+  host:
+    - python >=2.7
+    - pip
+  run:
+    - python >=2.7
+  test:
```

Comment:
I think this is an invalid subsection to requirements. Try to put a `requires` subsection in `test`.

Reply:
thanks a lot for the detailed instruction! just realize the test modules are not even packaged so i'll omit the test section until the next release :)